streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregate.java

```diff
@@ -66,11 +67,14 @@ public void enableSendingOldValues() {
         private WindowStore<K, T> windowStore;
         private TupleForwarder<Windowed<K>, T> tupleForwarder;
         private StreamsMetricsImpl metrics;
+        private InternalProcessorContext context;
```

Comment:
The super class `AbstractProcessor ` has a `ProcessorContext context`, so maybe rename it to `internalContext`?